phys2denoise/metrics/chest_belt.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+from scipy.interpolate import interp1d
+
+
+def rvt(belt_ts, peaks, troughs, samplerate, lags=(0, 4, 8, 12)):
```

Comment:
@smoia do we want lags (and/or downsampling) in these functions? This also comes up in #19.

Reply:
Maybe as optional steps rather than mandatory, but sure, why not?
Unless you think those steps would be better suited for other places in the code (if they apply to many or most of the metrics, and they're not metric-specific like RETROICOR, for instance).
What did you have in mind (we can also discuss it in the upcoming meeting if you think it's worth it)?

Reply:
The reason I was thinking we might want them separate is because adding lags is fairly generic and could probably be separated into its own function and performed on all metrics at once.

Reply:
I perfectly agree! Could you two work together on moving the lagging and resampling in a `utils` script?

Reply:
I think there's already actually a utils lagging function in #19 so I could probably refactor once everything gets merged together? Either that or I can copy/paste from there and pray for no merge conflicts.